Podfile

```diff
@@ -7,10 +7,16 @@ target 'Cineaste App' do
   pod 'SwiftLint'
   pod 'NearbyMessages'
   pod 'Kingfisher'
+  pod 'SwiftMonkeyPaws', '~> 2.1.0'
```

Comment:
I think this should only be in `Debug` builds, like so:

```ruby
  pod 'SwiftMonkeyPaws', :configuration => 'Debug'
```

Reply:
Nice tip! Didn't know that this is possible